src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figures {
+    String toString();
+
+    double area();
```

Comment:
```suggestion
    double getArea();
```
по конвенції для методів зазвичай використовують дієслова
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names